src/migratesettings.h

```diff
@@ -19,4 +19,10 @@ void migrate_settings()
 		g_settings->setBool("touch_gui", value);
 		g_settings->remove("enable_touch");
 	}
+
+	// Disables anticheat
+	if (g_settings->existsLocal("disable_anticheat")) {
+		g_settings->setU16("anticheat_flags", 0);
```

Comment:
existsLocal returning true doesn't mean the setting is set to true

Reply:
Oh, mistaked